### ISSUE TITLE
maintained

### ISSUE THREAD

**Issue by basz** [djmaze/dobs-volume-plugin#1]
via via via, i got here.

exited to use this plugin, but it seems to fail on first try.

```
docker run -it --rm -v my-volume:/mnt alpine
docker: Error response from daemon: open /var/lib/docker/plugins/f13ccd322907a5fdea57145f5a2bd144703be91407a82abd8c3f2754c8df8f0d/propagated-mount/my-volume/data: no such file or directory.
```

curious to know if this is still maintained? thank you!

**Comment by djmaze** [comment]
Hi, thanks for your feedback. Interesting to know that other people are trying this out ;)

The state is "works for me", so there were no changes in the last years.

I guess there is a small hurdle for usage this which is not documented: you need to create the initial `data` subdirectory in the volume before first trying to mount it via this driver. There is now an issue for this (#2).

**Comment by basz** [comment]
Aha, I see. I did try to create it in the local dir... thx will try that.

> Hi, thanks for your feedback. Interesting to know that other people are trying this out ;)

Well you are the only one left standing i guess :-)

**Comment by basz** [comment]
yep, that worked! great.

I'm using using this on a small (for now single node) swarm... Am I correct in assuming a volume needs to be created on each node of the swarm?

**Comment by djmaze** [comment]
No, you just need to create the volume on one node and it will be propagated. If you use `docker stack deploy` (recommended), it is sufficient to just define the volume in the yaml.
